optaweb-vehicle-routing-benchmark/README.adoc

```diff
@@ -0,0 +1,31 @@
+= OptaWeb Vehicle Routing Benchmark
+
+Measures distance matrix calculation for N locations using GraphHopper.
+
+.Prerequisites
+* The Belgium OSM file must exist in `local/openstreetmap/belgium-latest.osm.pbf` relative to the working directory.
+
+== Usage
+
+[source]
+----
+java [THREAD_COUNT] -jar target/optaweb-vehicle-routing-benchmark-VERSION-exec.jar [LOCATION_COUNT] [OUTPUT_FILE_NAME]
```

Comment:
The [THREAD_COUNT] argument is in my opinion misleading; the reader learns it's a system property in fact just in the middle of the text below, while the other arguments are just simple options.

Reply:
Yes. It's a bit different from the other ones but on conceptual level it's correct. In my opinion, it makes the usage example easier to read and understand.

I considered
```
java [-Djava.util.concurrent.ForkJoinPool.common.parallelism=THREAD_COUNT] \
-jar target/optaweb-vehicle-routing-benchmark-VERSION-exec.jar [LOCATION_COUNT] [OUTPUT_FILE_NAME]
```
too long.

Reply:
Wdyt?

Reply:
I don't think it's too long, but it's very clear regarding in what format is a user supposed to pass the option.

If you want to keep the original notation, let's mention it's this system property right on the first line of the argument's description.